examples/embedfs/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/donseba/go-htmx/examples/embedfs
+
+go 1.22
+
+require github.com/donseba/go-htmx v1.11.3
+
+replace github.com/donseba/go-htmx => ./../../
```

Comment:
since there's no tagged version I did this to show what it looks like, would have to remove and set to the correct version if this looks good :)